Dockerfile.rhel

```diff
@@ -1,47 +1,40 @@
+ARG BUILD_DIR=/tmp/service-binding-operator
+
 # On CI, use
 # FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
 
 FROM openshift/origin-release:golang-1.13 AS builder
+ARG BUILD_DIR
 
 ENV LANG=en_US.utf8
 ENV GIT_COMMITTER_NAME devtools
 ENV GIT_COMMITTER_EMAIL devtools@redhat.com
 LABEL com.redhat.delivery.appregistry=true
 
-WORKDIR /go/src/github.com/redhat-developer/service-binding-operator
-
-# Copy only relevant things (instead of all) to speed-up the build.
-COPY assets assets
-COPY build build
-COPY cmd cmd
-COPY deploy deploy
-COPY hack hack
-COPY manifests manifests
-COPY pkg pkg
-COPY test test
-COPY vendor vendor
-COPY go.mod .
-COPY go.sum .
-COPY LICENSE .
-COPY Makefile .
-COPY tools.go .
+# GOPATH-ish dir structure not needed anymore
+WORKDIR $BUILD_DIR
```

Comment:
Since the path is fixed, how about setting the value as `/tmp/service-binding-operator`

Reply:
does this help?
`ARG BUILD_DIR=/tmp/service-binding-operator`